.npmrc

```diff
@@ -0,0 +1 @@
+registry=https://registry.npmjs.org
```

Comment:
why do we see this?

Reply:
I use an own registry and when i do an simple `npm publish` it will be published to my registry and not not npmjs.org.
So we better specify it in .npmrc

Reply:
I have cloned the master branch, make my changes and then maked a pull request for the dev branch. If this was wrong i can change it.

Reply:
@rottmann Yes I know, the question was why does it appear in the PR.

@tommy87 yes that is the issue.